src/views/calendar.hbs

```diff
@@ -2,5 +2,6 @@
 <title>{{{__ "calendar_title"}}} – {{{__ "common_title"}}}</title>
 {{/section}}
 
-<iframe width="100%" height="100%" frameborder="0" src="https://teamup.com/ksb15466a44b299f51"
-  style="overflow: hidden"></iframe>
+<div class="embed-responsive embed-responsive-4by3" style="height: 100%;">
```

Comment:
Is it still necessary to add `style="height: 100%;"`?

Reply:
Yes, makes the difference between
![image](https://user-images.githubusercontent.com/34481945/71544032-83ab1e00-297a-11ea-8bf9-2ad74862545c.png) and ![image](https://user-images.githubusercontent.com/34481945/71544042-94f42a80-297a-11ea-8161-58c3dae02581.png)

Reply:
There is an existing bootstrap util class already: `.h-100`

Reply:
Thanks, I changed it